tiledb/sm/group/group.cc

```diff
@@ -527,13 +512,12 @@ const Config& Group::config() const {
   return config_;
 }
 
-Status Group::set_config(Config config) {
+void Group::set_config(Config config) {
   config_ = config;
-  return Status::Ok();
 }
 
-Status Group::clear() {
-  return group_details_->clear();
+void Group::clear() {
+  group_details_->clear();
```

Comment:
Comment: This function is a telltale that `Group` does not do RAII right.

Reply:
The primary problem I see is that `close` flushes the metadata, which could be the job of a dedicated function. I opened SC-26527 to track this.